### PR TITLE
Update the bundled version of makeself used to create static builds.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

This syncs our bundled version of makeself with [the latest upstream release](https://github.com/megastep/makeself/releases/tag/release-2.5.0), as we’ve been using a version that is almost six years out of date. The update should significantly improve compatibility on more exotic Linux systems.

It also updates the metadata we embed in the archive itself to better reflect the current state of the project.

##### Test Plan

Testing consists of creating static builds locally from this PR and verifying that they install correctly.

##### Additional Information

Fixes: https://github.com/netdata/netdata/issues/9517 (probably), and likely a number of other issues related to static installs.

### CODE REVIEW

File: packaging/makeself/makeself.sh
Comment (thiagoftsm) [inline packaging/makeself/makeself.sh:124]: @Ferroin , please, take a look in this warning from `shellcheck`.

Comment (thiagoftsm) [reply packaging/makeself/makeself.sh:124]: I tested locally the static binary build process and also the normal installation and everything worked. I am only waiting for these warnings to be addressed before to approve.

Comment (Ferroin) [reply packaging/makeself/makeself.sh:126]: Should be resolved now.

**Review by thiagoftsm — DISMISSED** [review]
Last commit did not modify code already tested and considering local tests were successful, LGTM!

**Comment by tkatsoulas** [comment]
I am facing an issue, netdata service is not starting/ exits, investigating it further. 
```
Ooops! it seems netdata is not started.
Attempting another netdata start using command 'systemctl start netdata'
[/opt/netdata]# systemctl start netdata 
 OK  ''

Hm... it seems netdata is still not started.
 WARNING  Netdata service still not started, attempting another forced restart by running '/opt/netdata/bin/netdata '

[/opt/netdata]# stop_all_netdata 
 OK  ''

[/opt/netdata]# /opt/netdata/bin/netdata 
2023-04-06 13:08:46: netdata INFO  : MAIN : CONFIG: cannot load user config '/opt/netdata/etc/netdata/netdata.conf'. Will try the stock version.
2023-04-06 13:08:46: netdata INFO  : MAIN : CONFIG: cannot load stock config '/opt/netdata/usr/lib/netdata/conf.d/netdata.conf'. Running with internal defaults.
2023-04-06 13:08:46: netdata INFO  : MAIN : CONFIG: cannot load cloud config '/opt/netdata/var/lib/netdata/cloud.d/cloud.conf'. Running with internal defaults.
2023-04-06 13:08:46: netdata FATAL : netdata : Cannot change directory to '/opt/netdata/var/log/netdata' # : Invalid argument

2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: initializing shutdown with code 1...
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: next: create shutdown file
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, create shutdown file - next: disable maintenance, new queries, new web requests, new streaming connections and aclk
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, disable maintenance, new queries, new web requests, new streaming connections and aclk - next: stop replication, exporters, ML training, health and web servers threads
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop replication, exporters, ML training, health and web servers threads - next: stop collectors and streaming threads
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop collectors and streaming threads - next: stop ML prediction and context threads
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop ML prediction and context threads - next: stop maintenance thread
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop maintenance thread - next: clean rrdhost database
2023-04-06 13:08:46: netdata INFO  : MAIN : RRD: cleaning up database [0 hosts(s)]...
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, clean rrdhost database - next: prepare metasync shutdown
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, prepare metasync shutdown - next: stop aclk threads
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop aclk threads - next: stop all remaining worker threads
2023-04-06 13:08:46: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, stop all remaining worker threads - next: cancel main threads
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[timex]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[idlejitter]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): HEALTH
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): STATS_GLOBAL
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): STATS_WORKERS
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): STATS_SQLITE3
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): PLUGINSD
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): SERVICE
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): STATSD_FLUSH
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): EXPORTING
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): ACLK_MAIN
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): RRDCONTEXT
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): REPLAY[1]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[tc]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[diskspace]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[proc]
2023-04-06 13:08:46: netdata INFO  : MAIN : EXIT: No thread running (marking as EXITED): P[cgroups]
2023-04-06 13:08:46: netdata INFO  : MAIN : Waiting 17 threads to finish...
2023-04-06 13:08:47: netdata INFO  : MAIN : All threads finished.
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: in     100 ms, cancel main threads - next: close SQL context db
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, close SQL context db - next: closed SQL main db
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, closed SQL main db - next: free openssl structures
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, free openssl structures - next: remove incomplete shutdown file
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: in       0 ms, remove incomplete shutdown file - next: exit
2023-04-06 13:08:47: netdata INFO  : MAIN : NETDATA SHUTDOWN: completed in 101 ms - netdata is now exiting - bye bye...
 FAILED  ''


  ^
  |.-.   .-.   .-.   .-.   .-.   .  netdata  .-.   .-.   .-.   .-.   .-.   .-
  |   '-'   '-'   '-'   '-'   '-'               '-'   '-'   '-'   '-'   '-'   
  +----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+--->

[/opt/netdata]# chmod 0644 /opt/netdata/etc/netdata/netdata.conf 
 OK  ''

```

**Comment by Ferroin** [comment]
@tkatsoulas The issue you saw should be resolved now.

**Review by tkatsoulas — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**
